web/inspector.css

```
@@ -375,3 +375,34 @@ body {
     height: 100%;
     background-color: #181818;
 }
+
+.lump-view-playpal {
```

Most other views have a 64px top margin to keep them from knocking right up onto the header above

```suggestion
.lump-view-playpal {
    margin-top: 64px;
```